### PR TITLE
Authentication tests with postman examples.

### PR DESCRIPTION
**Pull request by Delnote**
Authentication tests with postman examples.

### CODE REVIEW

**Comment by JDIReporter** [comment]
[170] - https://transfer.sh/oQryB/jdi-dark-tests.tar.gz

**Comment by JDIReporter** [comment]
[170] - https://transfer.sh/N7zOD/jdi-dark-bdd-tests.tar.gz

**Comment by JDIReporter** [comment]
[170] - Allure report on Netlify: https://5e37c090e9be2030e0746884--jdi-dark.netlify.com

**Comment by JDIReporter** [comment]
[172] - https://transfer.sh/4jkos/jdi-dark-tests.tar.gz

**Comment by JDIReporter** [comment]
[172] - https://transfer.sh/QMyms/jdi-dark-bdd-tests.tar.gz

**Comment by JDIReporter** [comment]
[172] - Allure report on Netlify: https://5e37c39149a8d8513201037f--jdi-dark.netlify.com

File: jdi-dark-tests/src/test/java/com/epam/jdi/httptests/PostmanAuthTests.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        resp.assertThat()
+                .statusCode(HttpStatus.SC_UNAUTHORIZED)
+                .body("", equalTo(body));
+    }
+
+    @Test
+    public void oauthTest() {
+        String key = "RKCGzna7bv9YD57c";
+        String nonce = "R6MyHe5WCRx";
+        Timestamp timestamp = new Timestamp(System.currentTimeMillis());
```
Comment (ElenaPoduzova) [inline jdi-dark-tests/src/test/java/com/epam/jdi/httptests/PostmanAuthTests.java]: Please fix codacy issue here: timestamp is not used.

**Comment by JDIReporter** [comment]
[177] - https://transfer.sh/xbWAy/jdi-dark-tests.tar.gz

**Comment by JDIReporter** [comment]
[177] - https://transfer.sh/kn9OX/jdi-dark-bdd-tests.tar.gz

**Comment by JDIReporter** [comment]
[177] - Allure report on Netlify: https://5e37e8749fec2e93eac4a3ed--jdi-dark.netlify.com

**Pull request merged**